.devcontainer/.zshrc

```diff
@@ -0,0 +1,104 @@
+# If you come from bash you might have to change your $PATH.
+# export PATH=$HOME/bin:/usr/local/bin:$PATH
+
+# Path to your oh-my-zsh installation.
+export ZSH="$HOME/.oh-my-zsh"
```

Comment:
I'm not sure how i feel about something like this. How does this impact someone who doesnt have oh-my-zsh installed, or any of the themes/plugins here.  

Same with the aliases, what if we're overriding some aliases people already have and would rather not change?

Reply:
There's no conflict here because it's a container specifically for dev